content/en/docs/guides/retry.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Retry 
+description: >-
+  gRPC takes the stress out of failures! Get fine-grained retry control and detailed insights with OpenCensus and OpenTelemetry support.
+---
+
+### Overview
+
+Retries are a key pattern for making services more reliable. By re-attempting failed operations, applications can overcome temporary issues like network glitches or server timeouts. This is essential for modern cloud applications to handle the inevitable transient faults that occur.
+
+For the best practice, applications should understand what failed operation is suitable for retry, define exponential backoff parameters for retry delay, determine the number of retry attempts, and also monitor retry metrics.
+
+gRPC takes the stress out of failures with retry policies support, default transparent retry and detailed insights with OpenCensus and OpenTelemetry support.
+
+### How gRPC client retry works
+
+Once enabled (transparent retry is enabled by default, see later sections), gRPC's built-in retry logic saves the call history for potential retries and monitors RPC events. Note that 'retry' means creating a new transport stream when a previous connection attempt failed and replaying the saved calls on the newly created stream.
```

Comment:
Toying with language

> gRPC’s built-in retry logic saves the call's history for potential retries and monitors RPC events. Even if there is no retry policy configured, gRPC still saves the call's history in case it needs to perform transparent retry (discussed in a later section). Note that ‘retry’ means replacing a failed call with a new call and replaying the call's history on that newly created call.